pkg/controller/operators/olm/operator.go

```diff
@@ -1366,6 +1367,14 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 			return
 		}
 
+		// Check if Webhooks have valid rules
+		for _, desc := range out.Spec.WebhookDefinitions {
+			if syncError = install.ValidWebhookRules(desc.Rules); syncError != nil {
```

Comment:
future nit: we should tie these validations to the the validation library used in opm